util/types/json/serdes.go

```diff
@@ -93,6 +93,43 @@ import (
                              // lengths up to 16383, and so on...
 */
 
+const (
```

Comment:
The constant 4 is used somewhere else in this file, we could use a const variable for all of them.